Silicon/Apple/T602XFamilyPkg/T602XFamilyPkg.dsc.inc

```diff
@@ -48,6 +47,8 @@
   #all Rhodes chips have three clusters, "boot", P-core/E-core clusters
   # for multi-die systems, this cluster setup is duplicated on the second die
   gAppleSiliconPkgTokenSpaceGuid.PcdClusterCount|3|UINT32|0x00004407
+  gAppleSiliconPkgTokenSpaceGuid.PcdBootArgsPointer|0x10A00000000
```

Comment:
for t6020, the fixed address is 0x10220000000, not 0x10A00000000, that needs to change before i can merge the PR